Assets/Scripts/Framework/PopupManegement/InteractablePopupTriggerActivator.cs

```diff
@@ -0,0 +1,9 @@
+using DerailedDeliveries.Framework.Gameplay.Interactions;
+
+namespace DerailedDeliveries.Framework.PopupManagement
+{
+    public class InteractablePopupTriggerActivator : PopupTriggerActivator<Interactable>
```

Comment:
```suggestion
    public class InteractablePopupTriggerActivator : PopupTriggerActivator<Interactable> {}
```
Also missing (inherit doc?) summery.